### PR TITLE
Bug: About page missing

### PR DESCRIPTION
**Pull request by CIMAManuel**
Add ABOUT title and sub menu for the page.
Add for any section a "back to top" link.

### CODE REVIEW

File: frontend/about.html [hunk 1]
```diff
@@ -109,74 +109,135 @@ <h2 class="page_title ">ABOUT</h2>
...
+            <p>In 2011, the <a href="https://www.gfdrr.org" target="_blank">Global Facility for Disaster Reduction and Recovery</a> (GFDRR) launched the Open Data for Resilience Initiative to apply the concepts of the global open data movement to the challenges of reducing vulnerability to natural hazards and the impacts of climate change. OpenDRI supports <a href="http://www.worldbank.org/en/topic/disasterriskmanagement" target="http://www.worldbank.org/en/topic/disasterriskmanagement">World Bank Regional Disaster Risk Management Teams</a> to build capacity and long-term ownership of open data projects with client countries that are tailored to meet specific needs and goals of stakeholders.</p>
+
+            <!-- Back to the top -->
+            <div class="text-center">
+                <a href="#startReading"><span class="small"><span class="small text-muted"><i class="fa fa-arrow-up" aria-hidden="true"></i> top</span></span></a>
```
Comment (oncletom) [inline frontend/about.html]: The [HTML5 specification](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-href) states that using `#top` or `#` actually directs the user to the top of the document.

The `startReading` id can be removed and the `href` can be updated to whichever of these values.

File: frontend/about.html [hunk 2]
```diff
@@ -109,74 +109,135 @@ <h2 class="page_title ">ABOUT</h2>
...
+                <mainmenu ></mainmenu>
+            </div>
 
-            <mainmenu ></mainmenu>
         </div>
 
+        <div class="row" id="top-menu">
+            <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
+                <!--<hazardmenu></hazardmenu>-->
+                <h1 class="page_title">ABOUT</h1>
```
Comment (oncletom) [inline frontend/about.html]: In #219, @pzwsk suggested a different page title: "About this website".

**Comment by oncletom** [comment]
Hey @dedandy and @CIMAManuel, it becomes a bit annoying things get merged without a proper review. I have flagged that when I was in Savona. I flagged this when #243 got merged. It happens again with this pull request.

I have 2 other concerns in regard of this pull request:

1. the sole expectation of #219 does not match
2. it's nice to add a table of contents but it's not what has been requested in #219

**Comment by dedandy** [comment]
sorry, How can come back?  I wrong all, I lose all my latest work...

**Comment by oncletom** [comment]
I guess opening a new pull request with the correct title will be enough.

When you write "my latest work", do you refer to #243? If so, you still have the branch on your laptop. If not, I can see 2 solutions:

- checkout the `origin/fixing_frontend_task` (do a `git fetch` first, I restored the branch).
- browse the Git History of the repo, the 2 commits "Merge branch 'master' into fixing_frontend_task" (04ffe3d6582a0826fe6966bbe38e0da62def1248) and "Rework easy simple email form following …" (15f08701c5e2b0819a73ff75ff9db6d05927cc4d) are still around and contain the work.

**Comment by CIMAManuel** [comment]
@oncletom I'll open a new pull request for this issue, ok?

**Comment by dedandy** [comment]
yes, #243 ... I found the branch on laptop. It is ok, I found all my modify. 👍 
Now I try to make 1 branch 1 commit  and 1 push for any single issue .... thx

**Pull request merged**